website/src/client/utils/constructAppetizeURL.tsx

```diff
@@ -40,6 +40,7 @@ export default function constructAppetizeURL({
 
   const appetizeKey = constants.appetize.public_keys[previewQueue][platform];
   const appParams = {
+    EXDevMenuDisableAutoLaunch: true,
```

Comment:
What does this do when using Expo Go on our default Appetize account? When is needed?

Reply:
Correct me if I'm wrong @esamelson, but I don't think this does anything in Expo Go. It uses a different parameter to skip the welcome menu.

We need this when loading a devclient in Appetize, something we intend to do in the near future. This basically also skips the onboarding menu when opened for the first time.